### PR TITLE
Fixed warnings on gulp build

### PR DESCRIPTION
**Pull request by ms-mrigoli**
## Background
_There were some SassDoc warnings when building the project._

## Changes done
- Fixed SassDoc comments which caused the warnings.

## Notes
The warnings were:
```
» [WARNING] @parameter must at least have a name. Location: _functions.scss:100:102
» [WARNING] Annotation `parameter` is not allowed on comment from type `css` in `_buttons.scss:56:70`.
» [WARNING] Annotation `parameter` is not allowed on comment from type `css` in `_buttons.scss:56:70`.
» [WARNING] Annotation `parameter` is not allowed on comment from type `css` in `_buttons.scss:56:70`.
» [WARNING] Annotation `parameter` is not allowed on comment from type `css` in `_buttons.scss:56:70`.
» [WARNING] Annotation `parameter` is not allowed on comment from type `css` in `_buttons.scss:56:70`.
» [WARNING] Item `_sc-from-hex` requires `_hex-to-dec` from type `function` but this item doesn't exist.
```
Which ~~absolutely does not~~ has ~~anything~~ to do with my colaboration with the mixin `c-mobile-menu-button()` which I've added recently... At least not all warnings!

### CODE REVIEW

File: assets/styles/modules/_buttons.scss [hunk 1]
```diff
@@ -69,10 +69,8 @@ $c-button-border-radius: toem(5px) !default;
 /// </div>
 /// @todo Maybe adding different animations?
 
-///
-$mobile-menu-button-transition: $msuxf-transition-duration + ' ' + $msuxf-transition-timing !default;
-
 @mixin c-mobile-menu-button($width, $height, $stroke, $color) {
+  $mobile-menu-button-transition: $msuxf-transition-duration + ' ' + $msuxf-transition-timing !default;
```
Comment (arzafran) [inline assets/styles/modules/_buttons.scss]: the ```///``` where there to document the parameter when running sassdoc, I'm not sure if they work inside a mixin, but can you explain a little why this change is needed?

Comment (ms-mrigoli) [reply assets/styles/modules/_buttons.scss]: The variable between the SassDoc comment and the @mixing declaration caused these warnings:

```
» [WARNING] Annotation `parameter` is not allowed on comment from type `css` in `_buttons.scss:56:70`.
» [WARNING] Annotation `parameter` is not allowed on comment from type `css` in `_buttons.scss:56:70`.
» [WARNING] Annotation `parameter` is not allowed on comment from type `css` in `_buttons.scss:56:70`.
» [WARNING] Annotation `parameter` is not allowed on comment from type `css` in `_buttons.scss:56:70`.
» [WARNING] Annotation `parameter` is not allowed on comment from type `css` in `_buttons.scss:56:70`.
``` 

Maybe the best way to deal with it is to put the variable with the `///`comment.

Comment (ms-mrigoli) [reply assets/styles/modules/_buttons.scss]: Yes, that was it, I've moved the variable declaration, It was a mistake to put it within the `@mixin`. Thank you for noticing it.

Comment (arzafran) [reply assets/styles/modules/_buttons.scss]: I think the issue is with the previous line, the `@todo` can't be parsed and fails. (its pointing to line :70 but)

Comment (ms-mrigoli) [reply assets/styles/modules/_buttons.scss]: Nope, I've just tested to put the var where it was removing the `@todo` and it shows the warnings. 

The problem seems to be to put the variable within the SassDoc mixin declaration and the mixin declaration. Which makes sense.
